src/com/sun/ts/tests/jms/ee20/resourcedefs/annotations/JspClient.jsp

```diff
@@ -23,7 +23,7 @@
 <%@ page import="java.util.Enumeration" %>
 <%@ page import="java.util.Properties" %>
 <%@ page import="java.util.ArrayList" %>
-<%@ page import="javax.transaction.*" %>
+<%@ page import="jakarta.transaction.*" %>
```

Comment:
My first question is whether we need a javax.transaction.xa class or some jakarta.transaction class.  I could be overlooking something but I don't see any javax.transaction or jakarta.transaction classes referenced in JspClient.jsp.